Solutions/Authomize/Analytic Rules/Chain_of_3_or_more_roles.yaml

```diff
@@ -0,0 +1,59 @@
+id: 25bef734-4399-4c55-9579-4ebabd9cccf6
+name: Chain of 3 or more roles
```

Comment:
Rule name should be self-explanatory. This is very open ended and hard to interpret.

Reply:
updated

Reply:
I think the Name is still not suitable. Please improve.

Reply:
Improved - used a combo of the description - should be more understandable